seed/utils/inventory_filter.py

```diff
@@ -49,6 +50,15 @@ def get_filtered_results(request: Request, inventory_type: Literal['property', '
             'results': []
         })
 
+    if ids_only and (per_page or page):
+        return JsonResponse({
+            'success': False,
+            'message': 'Cannot pass query parameter "ids_only" with "per_page" or "page"'
+        }, status=status.HTTP_400_BAD_REQUEST)
+
+    page = page or 1
+    per_page = per_page or 1
```

Comment:
assign default values to page and per_page after evaluating if the correct query params were passed